media/css/base/banners/firefox-newsletter.scss

```
@@ -0,0 +1,148 @@
+
+        &::before {
+            @include bidi(((left, 0, right, 0),));
+            background: transparent url('/media/img/banners/fx-newsletter/lock-sticker.svg') center center no-repeat;
+            background-size: contain;
+            content: '';
+            display: block;
+            height: 60px;
+            position: absolute;
+            top: 0;
+            width: 60px;
+        }
+    }
+
+    .c-banner-cta {
+        margin: 0;
+    }
+
+    .c-banner-button {
+        background: $color-yellow-20 linear-gradient(to right,$color-yellow-20, $color-orange-50, $color-yellow-20, $color-yellow-20);
+        background-size: 300% 100%;
+        border: 0;
+        border-radius: $border-radius-sm;
+        color: $color-black;
+        padding: $spacing-sm $spacing-lg;
+        transition: all 300ms ease-out;
+        white-space: nowrap;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-position: 100% 0;
+            border-color: $color-black;
+            box-shadow: $box-shadow-sm;
+            color: $color-black;
+        }
+    }
+
+    // Close button
+    .c-banner-close {
+        @include image-replaced;
+        background: transparent url('#{$image-path}/icons/close-white.svg') center center no-repeat;
+        @include background-size(15px 15px);
+        border: none;
+        cursor: pointer;
+        display: block;
+        height: 21px;
+        min-width: 0;
+        padding: 0;
+        width: 21px;
+        position: absolute;
+        top: 8px;
+        right: 8px;
+
+        &:hover,
+        &:focus {
+            @include transform(scale(1.1));
+            @include transition(transform 0.1s ease-in-out);
+        }
+
+        &:focus {
+            outline: 1px dotted $color-white;
+        }
+
+        // hide the 'Close' text
+        span {
+            @include visually-hidden;
+        }
+    }
+
+
+    @media #{$mq-md} {
+        background-image: url('/media/img/banners/fx-newsletter/background-grid-desktop.svg');
+
+        .c-banner-copy {
+            align-items: center;
+            display: flex;
+            gap: $spacing-lg;
+            flex-flow: row nowrap;
+            justify-content: space-between;
+
+            &::before {
+                top: 50%;
+                margin-top: -30px;
+            }
+        }
+
+        .c-banner-title {
+            padding: 0;
+            margin: 0;
+        }
+
+        .c-banner-button {
+            width: auto;
+        }
+    }
+
+    @media (prefers-reduced-motion: reduce) {
+        .c-banner-button {
+            background-image: linear-gradient(to right,$color-yellow-20, $color-orange-50);
+            transition: background-color 300ms ease-out;
+
+            &:hover,
+            &:focus,
+            &:active {
+                background-image: linear-gradient(to right,$color-yellow-20, $color-yellow-20);
+            }
+        }
+    }
+}
+
+html[data-firefox-newsletter-banner-closed="true"] {
```

Would keep in document flow for the no-js possibility too

```suggestion
html[data-firefox-newsletter-banner-closed="true"], html.no-js {
```